src/rest-server/src/models/v2/storage.js

```diff
@@ -90,6 +90,12 @@ const convertVolumeDetail = async (pvc) => {
     if (pv.spec.flexVolume.options.mountoptions) {
       storage.mountOptions = pv.spec.flexVolume.options.mountoptions.split(',');
     }
+  } else if (pv.spec.csi) {
+    if (pv.spec.csi.driver === 'dshuttle') {
+      storage.type = 'dshuttle';
```

Comment:
update api docs accordingly

Reply:
DShuttle just used by internal users in this release. Will update API when DShuttle published